lab/utils/appointment_mail.py

```diff
@@ -1,42 +1,38 @@
 import re
-from typing import Optional
 
 from django.conf import settings
+from django.core.mail import get_connection
 from django.template import defaultfilters
 from django.utils.html import strip_tags
 from django.utils.safestring import mark_safe
 from django.utils.timezone import localtime
 import urllib.parse as parse
 
-from cdh.core.utils.mail import send_template_email
-
+from experiments.email import AppointmentConfirmEmail
 from experiments.models import Appointment, Experiment, TimeSlot
 from main.utils import get_supreme_admin
 from participants.models import Participant
 
 CANCEL_LINK_REGEX = r'{cancel_link(?::\"(.*)\")?}'
 
 
-def send_appointment_mail(appointment: Appointment) -> None:
+def send_appointment_mail(appointment: Appointment, override_content=None) -> None:
     experiment = appointment.experiment
     participant = appointment.participant
     time_slot = appointment.timeslot
 
-    admin = get_supreme_admin()
-    template = 'api/mail/new_appointment'
-
     subject = 'Bevestiging inschrijving experiment UiL OTS: {}'.format(
         experiment.name
     )
 
     replacements = {
-        '{experiment_name}': experiment.name,
-        '{experiment_location}': '',
-        '{participant_name}': participant.name,
-        '{leader_name}': experiment.leader.name,
-        '{leader_email}': experiment.leader.user.email,
-        '{leader_phonenumber}': experiment.leader.phonenumber,
-        '{all_leaders_name_list}': experiment.leader.name,
+        'experiment_name': experiment.name,
```

Comment:
This look a lot cleaner now

Reply:
not sure why, but this change disappeared from the merge